server/data/prisonApi.ts

```diff
@@ -8,14 +8,60 @@ interface CaseLoad {
   type: string
 }
 
+// Possibly not an exhaustive list
+type LocationType =
+  | 'AREA'
+  | 'ADJU'
+  | 'ASSO'
+  | 'SPOR'
+  | 'LOCA'
+  | 'STOR'
+  | 'CLAS'
+  | 'WORK'
+  | 'EXER'
+  | 'IGRO'
+  | 'RESI'
+  | 'ROOM'
+  | 'WING'
+  | 'MEDI'
+  | 'CELL'
+  | 'INTE'
+  | 'BOX'
+  | 'RTU'
+  | 'VIDE'
+  | 'TRAI'
+  | 'LAND'
+  | 'SPUR'
+  | 'HOLD'
+
+interface Location {
+  locationId: number
+  locationType: LocationType
+  description: string
+  agencyId: string
+  currentOccupancy: number
+  operationalCapacity?: number
+  locationPrefix: string
+  userDescription?: string
+}
+
 class PrisonApi extends RestClient {
   constructor(token: string) {
     super('HMPPS Prison API', config.apis.hmppsPrisonApi, token)
   }
 
   async getUserCaseLoads(): Promise<Array<CaseLoad>> {
-    return this.get({ path: '/users/me/caseLoads' }) as Promise<Array<CaseLoad>>
+    return this.get({ path: '/api/users/me/caseLoads' }) as Promise<Array<CaseLoad>>
```

Comment:
This one should be moved (in a different story) https://nomis-user-dev.aks-dev-1.studio-hosting.service.justice.gov.uk/swagger-ui/index.html#/
`/me/caseloads`. gives:
```{
  "username": "TESTUSER1",
  "active": true,
  "accountType": "GENERAL",
  "activeCaseload": {
    "id": "WWI",
    "name": "WANDSWORTH (HMP)"
  },
  "caseloads": [
    {
      "id": "WWI",
      "name": "WANDSWORTH (HMP)"
    }
  ]
}```

Reply:
Yep, I added ticket [INC-86](https://dsdmoj.atlassian.net/browse/INC-86) for this ☺️